receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -73,6 +80,20 @@ func (oci *Receiver) Export(tes agenttracepb.TraceService_ExportServer) error {
 		return err
 	}
 
+	messageChan := make(chan *data.TraceData, 64)
```

Comment:
How did you come to choosing this capacity? A value that you gave you some optimal throughput?

Reply:
Perhaps let's add a comment or make this a const for easier discoverability and document it too.